pages/vi/vi-vagrant.md

```diff
@@ -21,7 +21,9 @@ There are three command line programs you could use to interact with Vagrant CLI
 
 2. **Command Prompt**: There are two Command Prompt options on Windows: **Command Prompt** and **Command Prompt(Admin)**. You could use either of them to access directories and interact with Vagrant CLI. The only difference is their default directory: **Command Prompt(Admin)** has `C:\Windows\system32` as default directory and **Command Prompt** has `C:\Users\YOUR_USERNAME` as default directory.
 
-2. **Windows PowerShell**: Windows PowerShell is a default shell interface installed on your machine, its default directory is `C:\Users\YOUR_USERNAME`.
+3. **Windows PowerShell**: Windows PowerShell is a default shell interface installed on your machine, its default directory is `C:\Users\YOUR_USERNAME`.
+
+**NOTE**: PowerShell and Command Prompt users will have to substitute backslashes “\” with forward slashes “/”throughout these introductory steps as it pertains to file paths.                       Example: If you see an instruction saying `cd desktop/OLE/Planet` instead use `cd desktop\OLE\Planet`
```

Comment:
It should be "Substitute forward slashes with backslashes", not the other way around